src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json

```diff
@@ -135,5 +136,7 @@
     "sulu_admin.has_changed_warning_dialog_text": "Dieser Inhalt wurde zwischenzeitlich von einem anderen Benutzer geändert. Wenn der Inhalt gespeichert wird, gehen diese Daten verloren.",
     "sulu_admin.edit_profile": "Profil bearbeiten",
     "sulu_admin.no_permissions": "Keine Berechtigung",
-    "sulu_admin.no_options_available": "Keine Optionen verfügbar"
+    "sulu_admin.no_options_available": "Keine Optionen verfügbar",
```

Comment:
is this translation used anywhere?

Reply:
I just had to add a comma at the end :wink: It is used, but not in this PR.

Reply:
🤦‍♂